src/main/java/org/graylog/integrations/ipfix/codecs/IpfixCodec.java

```diff
@@ -93,6 +93,9 @@ protected IpfixCodec(@Assisted Configuration configuration, IpfixAggregator ipfi
         final List<String> customDefFilePathList = configuration.getList(CK_IPFIX_DEFINITION_PATH);
         final List<URL> filePaths = new ArrayList<>();
 
+        //infoElementDefs = Builder.build(InformationElementDefinitions.class)
```

Comment:
It looks like some extra commented out code was added in commit 584c4f1aadac4aa21aa74de6e9c1e77c0d4ba6b6. Can this be removed?